src/Dfe.PlanTech.Web/Views/Shared/Components/ButtonWithEntryReference.cshtml

```diff
@@ -21,15 +21,12 @@
                 <button-with-link model=@buttonWithLink />
                 break;
             }
-        case Question question:
+        default:
```

Comment:
IMO it would be better to be explicit here, and do something like `case Question _:`, then have a default later on where we log an error.

That being said, this is also fine so approved!

Reply:
Resolved in: [aa4c4f9](https://github.com/DFE-Digital/plan-technology-for-your-school/pull/339/commits/aa4c4f998832be7162c55ef4f9a977e0fd14ba75)